parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -173,7 +182,7 @@ write_files:
   content: !!binary |
     {{GetVariableProperty "cloudInitData" "kubeletMonitorSystemdService"}}
 
-{{- if NeedsContainerd}}
```

Comment:
curious: is this diff needed?

Reply:
most ignorance on the difference between {{-if}} vs {{if -}. I assume the else clause but just checking for my sanity.

Reply:
It mostly helps keep the generated diff cleaner